Cargo.toml

```diff
@@ -9,15 +9,13 @@ name = "uuid_macro"
 repository = "https://github.com/uuid-rs/uuid_macro"
 version = "0.1.0"
 
-[lib]
-proc-macro = true
 
-[dependencies.syn]
-version = "0.13"
-
-[dependencies.quote]
-version = "0.5"
-
-[dependencies.proc-macro-hack]
-version = "0.4"
+[dependencies]
+uuid_macro_impl = { path = "./uuid_macro_impl"}
+proc-macro-hack = { version = "0.4"}
+uuid = { git = "https://github.com/mokeyish/uuid.git", branch = "master" }
```

Comment:
@Dylan-DPC Because  [this](https://github.com/uuid-rs/uuid/pull/218) have't approved yet. Do you know why about this snapshot.
![image](https://user-images.githubusercontent.com/16131917/39685815-a21f197e-51f7-11e8-9eca-e06f666fa7e0.png)

Reply:
Yes i know. Just made that note so we remember to switch back to the crate once that is completed.

Reply:
the PR has been merged

Reply:
ok